namnder/Talmanspresidiet/body_en.md

```diff
@@ -0,0 +1,8 @@
+# Speakers’ Presidium
+
+The Speaker's Bureau is tasked with organizing the SM and VM held during the year. The Speaker's Bureau consists of 6 Speakers, Deputy Speakers, and Secretary.
+
+## Section meetings (SM) and Electoral meetings (VM)
+
+§3 of the [Data section's statutes](https://styrdokument.datasektionen.se/stadgar#3-sektionsm%C3%B6tet) (Translated)
+"At the SM, all regular section members have the right to attend, speak, propose and vote. The section's auditors according to §10.1 have the right to attend, speak and propose. Honorary members, alumni members and members of the THS board have the right to attend and speak. In addition, the SM can co-opt outsiders with the right to attend and possibly also the right of expression."
```

Comment:
I don't think it's that import to stress this part and quote it word-for-word - it would be better to give a short description of what actually happens there and why they're important fora for the Chapter